tests/io/vasp/test_sets.py

```diff
@@ -107,21 +107,21 @@ def test_sets_changed(self):
             with open(input_set) as file:
                 text = file.read().encode("utf-8")
                 name = os.path.basename(input_set)
-                hashes[name] = hashlib.sha1(text).hexdigest()
+                hashes[name] = hashlib.sha256(text).hexdigest()
```

Comment:
Replaced weak `sha1` with `sha256` to fix [S324](https://docs.astral.sh/ruff/rules/hashlib-insecure-hash-function/), I don't think this would break anything as this hash is just an internal sanity check, but still pin this for confirmation.